### ISSUE TITLE
Show image of map with location

### ISSUE THREAD

**Issue by jakeboyles** [kmoe/slack-pokebot#25]
I was able to use google maps static image api to send images over to slack so you dont have to pull up a map. I can submit a PR if you would like...
<img width="635" alt="screen shot 2016-07-28 at 6 49 57 pm" src="https://cloud.githubusercontent.com/assets/2816632/17232177/3aa37cb0-54f4-11e6-9794-517ea776063c.png">

**Comment by kmoe** [comment]
Good idea - I'm currently changing the way the Slack messages are sent and may use a thumbnail for the Pokemon image, so I'll see if it's also possible to attach a map image. I'd like to prioritise having the Pokemon image there.

**Comment by jakeboyles** [comment]
@kmoe Yeah I incorporated your new way of using attachments. You can actually send up to 25 attachments to slack.
